sensor/common/sensor/central_sender_impl.go

```diff
@@ -112,10 +114,10 @@ func (s *centralSenderImpl) send(stream central.SensorService_CommunicateClient,
 			}
 
 			if msg.GetEvent().GetSynced() != nil {
-				log.Info("Sending synced signal to Central")
-				if s.onSync != nil {
-					go s.onSync()
```

Comment:
By looking at this diff, the `onSync` function seems not to be used anymore. Could we remove it maybe?